cmd/natc/natc.go

```diff
@@ -70,6 +71,17 @@ func main() {
 		log.Fatalf("site-id must be in the range [0, 65535]")
 	}
 
+	var ignoreDstTable *bart.Table[bool]
+	if *ignoreDstPfxStr != "" {
+		for _, s := range strings.Split(*ignoreDstPfxStr, ",") {
+			if s != "" {
+				if ignoreDstTable == nil {
+					ignoreDstTable = make(bart.Table[bool])
+				}
+				ignoreDstTable.Insert(netip.MustParsePrefix(strings.TrimSpace(s)), true)
```

Comment:
oh we should probably also check if the `pfx.Masked() != pfx` here like we do below